app/models/ability.rb

```diff
@@ -8,7 +8,10 @@ def initialize(user)
       if user.admin?
         can :manage, :all
       else
-        can :manage, :all
+        can :read, JobOpening
```

Comment:
This is looking much better now.  Non-super users have specific actions that they can do on certain resources instead of being able to manage all.  After all, you don't want regular users to create `JobOpening` entries, but you do want them to be able to see them.